rep-2001.rst

```diff
@@ -38,26 +38,45 @@ We define three main entry points for ROS users.
  * desktop (recommended)
  * ros_base
  * ros_core
+ * ros_minimal
```

Comment:
Note for final version: This REP is for "ROS Bouncy and newer" So most likely the final PR should be creating a new REP for the specific distros it targets (e.g. Foxy and newer)

---

Nit: wording above will need to be updated in the final version `We define ~~three~~ four main entry points`

Reply:
Oh, yes - I'm definitely not interested in trying to build this variant for Bouncy or Crystal